drivers/interrupt_controller/intc_vim.c

```diff
@@ -65,6 +65,10 @@ void z_vim_irq_init(void)
 		 "(%" PRIu32 ") interrupts",
 		 CONFIG_NUM_IRQS, num_of_irqs);
 	LOG_DBG("VIM: Number of IRQs = %u\n", num_of_irqs);
+
```

Comment:
I'd add a quick comment here, something like:

`/* Start with all interrupts masked */`

Other than that, LGTM.